oops_program_1.cpp

```diff
@@ -0,0 +1,58 @@
+#include<iostream>
+Class student
+{
+   Protected:
+       int roll_number;
+   public:
+       void get_number(int);
+       void put_number(void);
+ }
+Void student::get_number(int a)
+{
+    Roll_number=a;
+}
+Void student::put_number()
+{
+     Cout<<”Roll Number:”<<roll_number<<”\n”;
+}
+Class test:public student        // derivation//
+{
+       protected:
+             float sub1;
+             float sub 2;
```

Comment:
Does this represent a subject or the mark the student has obtained in the subject?